### DISCUSSION TITLE
email configuration

### DISCUSSION THREAD

**Discussion by asifrana1972** [horilla-opensource/horilla#200] [General]
Hi, Facing problem to connect mail server, How Gmail as connected with horilla.

**Comment by horilla-opensource** [comment]
> Hi, Facing problem to connect mail server, How Gmail as connected with horilla.

Hi @asifrana1972 ,
Did you use the app password or gmail password to the email configuration field?

**Reply by sharukhkhan234** [reply]
Hi!
How to Email Multiple Users at one time, in Horilla the provided option we can configure an email to a single user I think so, and I want to send the same email to multiple users at one time, check in the preview, I have provided you an SS.
![mail](https://github.com/horilla-opensource/horilla/assets/157000111/8c9c3d5d-2352-4200-83f5-8ae635e2adcf)

**Reply by horilla-opensource** [reply]
Hi @sharukhkhan234 ,
Once you have configured the email, you can bulk mail to employees from the employee view.
Go the employees menu, change to list view from the card view and select the employees for whom you want to send them mail. When the required employees are selected under the _Actions_ button, you'll find an option for _Bulk Mail_ and you can deliver mails to that employees.
Please refer the below screenshots:
1. 
<img width="1470" alt="Screenshot 2024-06-03 at 12 51 53 PM" src="https://github.com/horilla-opensource/horilla/assets/131998600/9872e2a0-1c54-4844-ad72-179a79267f7a">

2.
<img width="1470" alt="Screenshot 2024-06-03 at 12 49 43 PM" src="https://github.com/horilla-opensource/horilla/assets/131998600/178890af-785f-4676-b1e7-362cf3730afb">

3.
<img width="1470" alt="Screenshot 2024-06-03 at 12 50 54 PM" src="https://github.com/horilla-opensource/horilla/assets/131998600/627e1322-5338-4319-9683-ed27726cb125">

**Reply by sharukhkhan234** [reply]
can  you share how to do the configuration correctly, i configured it, and then tried to send an email, but it generated an error notification of
"Something went wrong". I tried but did not catch an error. Could you provide the right configuration procedure?

Regards!

**Reply by horilla-opensource** [reply]
Which email server are you trying to configure?

**Reply by sharukhkhan234** [reply]
Tried this one.
![mail_server](https://github.com/horilla-opensource/horilla/assets/157000111/5d14bea4-11ea-4e4f-a663-590e1463ab23)

**Reply by horilla-opensource** [reply]
Hi @sharukhkhan234 ,
Please set the _email host username_ with your gmail address and generate an app password for your gmail account and use that password for the _email authentication password_.
Please refer the below doc on how to generate app password.
[Sign in with app passwords](https://support.google.com/mail/answer/185833?hl=en)

**Reply by sharukhkhan234** [reply]
Thank you so much!
Successfully done

Regards,
